Week1/homework/js2-exercises/2-aboutMe.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>About Me</title>
+  </head>
+  <body>
+    <h1>About Me</h1>
+
+    <ul>
+      <li>Nickname: <span id="nickname"></span></li>
+      <li>Favorite food: <span id="fav-food"></span></li>
+      <li>Hometown: <span id="hometown"></span></li>
+    </ul>   
+	<script>
+		document.body.style.fontFamily=`Arial,sans-serif`;
+
+		document.querySelector('#nickname').innerText='David';
+		document.querySelector('#fav-food').innerText='Charcoal grilled meat';
+		document.querySelector('#hometown').innerText='London';
+
+		const items=document.querySelectorAll('ul li');
+		items.forEach(item=> {item.classList.add('list-item')})
```

Comment:
Good work. However good to keep in mind that this way of iterating is not supported by all browsers Here is more read about iteration options on nodeList 
https://css-tricks.com/snippets/javascript/loop-queryselectorall-matches/